README.md

```diff
@@ -36,11 +36,28 @@ for learning about the implementation of linear algebra routines.
 ```julia
 julia> using Gaius
 
-julia> blocked_mul(A, B) # multiply `A` and `B` and return the result
+julia> Gaius.mul!(C, A, B) # (multi-threaded) multiply A×B and store the result in C (overwriting the contents of C)
 
-julia> blocked_mul!(C, A, B) # multiply `A` and `B` and store the result in `C`
+julia> Gaius.mul(A, B) # (multi-threaded) multiply A×B and return the result
+
+julia> Gaius.mul_single_threaded!(C, A, B) # (single-threaded) multiply A×B and store the result in C (overwriting the contents of C)
+
+julia> Gaius.mul_single_threaded(A, B) # (single-threaded) multiply A×B and return the result
 ```
 
+Remember to start Julia with multiple threads with e.g. one of the following:
+- `julia -t auto`
+- `julia -t 4`
+- Set the `JULIA_NUM_THREADS` environment variable to `4` **before** starting Julia
```

Comment:
Interesting to note that I get similar performance with 18 and 36 threads on an 18 core/36 thread system.
BLAS normally suffers with too many threads, but the cache oblivious algorithm probably makes Gaius less sensitive.